spec/post_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations for Post model' do
+    before(:each) do
+      @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.',
+                          posts_counter: 0)
+      @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post', likes_counter: 0,
+                       comments_counter: 0)
+    end
+
+    it 'Title must not be blank' do
+      @post.title = nil
+      expect(@post).to_not be_valid
+
+      @post.title = 'Hello'
+      expect(@post).to be_valid
+    end
+
+    it 'Title must not exceed 250 characters' do
+      @post.title = 'less than 250 characters'
+      expect(@post).to be_valid
+    end
+
```

Comment:
- [ ] Good to add this test :+1: However to test if a post title must not exceed 250 characters, You have to create a post with title characters of more than 250 and expect the post to be invalid like this :point_down: 
  
  ```rb
  it 'Title must not exceed 250 characters' do
       @post.title = 'abc' * 100 ## post tile is more that 250 characters 
        expect(@post).to_not be_valid
  end
  ```